pkg/channel/http/http_channel.go

```diff
@@ -8,11 +8,15 @@ package http
 import (
 	"crypto/tls"
 	"fmt"
+	"strconv"
 	"strings"
 	"time"
 
 	"github.com/dapr/dapr/pkg/channel"
+	"github.com/dapr/dapr/pkg/config"
+	tracing "github.com/dapr/dapr/pkg/diagnostics"
 	"github.com/valyala/fasthttp"
+	"go.opencensus.io/trace"
```

Comment:
Can we find a way to encapsulate tracing inside the `diagnostics` package? I don't think we should get opencensus things inside our channels..

Reply:
tracing is a cross-cutting concern. We need to wrap tracing statements before and after the handler method. If we push this back to the diagnostics package as a callback, we'll need to reference channel package from the diagnostics package, which is backwards.

Reply:
There's no need to have diagnostics know about channels..

The code used by `go.opencensus.io/trace` uses simple primitives that can be passed to the method, without passing in any channel related things.

The code in this file is based on fasthttp request responses, things that the `diagnostics` package already knows.

Reply:
I can remove reference to opencensus.io/trace, but reference to tracing package itself remains.

Reply:
That's ok